redwood-tooling-codegen/src/main/kotlin/app/cash/redwood/tooling/codegen/protocolHostGeneration.kt

```diff
@@ -56,15 +54,15 @@ public class ExampleProtocolFactory<W : Any>(
   private val json: Json = Json.Default,
   private val mismatchHandler: ProtocolMismatchHandler = ProtocolMismatchHandler.Throwing,
 ) : GeneratedProtocolFactory<W> {
-  private val childrenTags: Map<WidgetTag, List<ChildrenTag>> = mapOf(
-        WidgetTag(1) to listOf(ChildrenTag(1)),
-        WidgetTag(3) to listOf(ChildrenTag(1)),
-        WidgetTag(1_000_001) to listOf(ChildrenTag(1), ChildrenTag(2)),
-        WidgetTag(1_000_002) to listOf(ChildrenTag(1)),
-      )
+  private val childrenTags: IntObjectMap<IntArray> = MutableIntObjectMap(4).apply {
+        put(1, intArrayOf(1))
+        put(3, intArrayOf(1))
+        put(1_000_001, intArrayOf(1, 2))
+        put(1_000_002, intArrayOf(1))
+      }
```

Comment:
Sent a CL for a builder API, but it'll be a while before we can use it: https://android-review.googlesource.com/c/platform/frameworks/support/+/3275076